swo/mpt/extensions/runtime/djapp/conf/default.py

```diff
@@ -165,6 +166,13 @@
     },
 }
 
+
+# OpenTelemetry configuration
+SERVICE_NAME = os.getenv("SERVICE_NAME", "Swo.Extensions.AdobeVIPM")
+USE_APPLICATIONINSIGHTS = os.getenv("USE_APPLICATIONINSIGHTS", "True").lower() in ("true", "1", "t")
+APPLICATIONINSIGHTS_CONNECTION_STRING = os.getenv("APPLICATIONINSIGHTS_CONNECTION_STRING", "")
+LOGGING_ATTEMPT_GETTER = os.getenv("LOGGING_ATTEMPT_GETTER", "adobe_vipm.utils.get_attempt_count")
```

Comment:
Do you think logging the number attempts is needed from the day 0 ?
When we switch from polling the API to get the events, to push the events to the extensions from the MPT platform, retries, delayed redeliveries etc should be controlled by the MPT platform.

If it is needed from the day 0 let's try to brainstorm if a better approach is possible.

Reply:
Yes, that's an important part since we have a not full cycle of processing like something failed and we reschedule the order somewhere in the middle. For sure in the future, it may be improved by the runtime. But we have what we have now :-)